main.go

```diff
@@ -364,7 +364,7 @@ func downloadReleaseAssets(assetRegex string, destPath string, githubRepo GitHub
 			assetPath := path.Join(destPath, asset.Name)
 			fmt.Printf("Downloading release asset %s to %s\n", asset.Name, assetPath)
 			if downloadErr := DownloadReleaseAsset(githubRepo, asset.Id, assetPath); downloadErr == nil {
-				fmt.Printf("Downloaded %s\n", assetPath)
+				fmt.Printf("\rDownloaded %s\n", assetPath)
```

Comment:
Its Carrier Return. It moves the cursor to point 0 and then writes from there. So it will overwrite the previous row.

Reply:
Right, it's just not obvious _why_ we'd need to return the cursor too point 0 here when looking at this code. I know, from the context of this PR, it's because you (might!) have the download progress to clean up... But outside the context of this PR, that won't be obvious. Moreover, download progress might be disabled if `withProgress` is `false`! So perhaps The `\r` should be written in `DownloadReleaseAsset` by the code that deals with writing the progress to screen?

Reply:
right, I just removed it instead.